samples/sampler/stories/default/Popup.js

```diff
@@ -25,36 +25,50 @@ export default {
 	component: 'Popup'
 };
 
-export const _Popup = () => {
-	const buttonsSelection = select('buttons', ['no buttons', '1 button', '2 buttons'], Config, 'no');
+export const _Popup = (args) => {
+	const buttonsSelection = args['buttons'];
 	const buttons = prop.buttons[buttonsSelection];
 
 	return (
 		<div>
 			<Popup
-				centered={boolean('centered', Config)}
-				closeButton={boolean('closeButton', Config)}
-				duration={select('duration', ['short', 'medium', 'long'], Config)}
-				noAnimation={boolean('noAnimation', Config)}
-				noAutoDismiss={boolean('noAutoDismiss', Config)}
+				centered={args['centered']}
+				closeButton={args['closeButton']}
+				duration={args['duration']}
+				noAnimation={args['noAnimation']}
+				noAutoDismiss={args['noAutoDismiss']}
 				onClose={action('onClose')}
 				onHide={action('onHide')}
 				onShow={action('onShow')}
-				open={boolean('open', Config)}
-				position={select('position', ['bottom', 'center', 'fullscreen', 'left', 'right', 'top'], Config)}
-				scrimType={select('scrimType', ['none', 'translucent', 'transparent'], Config, 'translucent')}
-				spotlightRestrict={select('spotlightRestrict', ['self-first', 'self-only'], Config, 'self-only')}
-				title={text('title', Config, 'Title')}
-				type={select('type', [null, 'fade', 'slide'], Config)}
+				open={args['open']}
+				position={args['position']}
+				scrimType={args['scrimType']}
+				spotlightRestrict={args['spotlightRestrict']}
+				title={args['title']}
+				type={args['type']}
 			>
-				<div>{text('children', Config, 'Hello Popup')}</div>
+				<div>{args['children']}</div>
 				{buttons}
 			</Popup>
-			Use KNOBS to interact with Popup.
+			Use CONTROLS to interact with Popup.
 		</div>
 	);
 };
 
+select('buttons', _Popup, ['no buttons', '1 button', '2 buttons'], Config, 'no');
```

Comment:
Please fix this line 
```suggestion
select('buttons', _Popup, ['no buttons', '1 button', '2 buttons'], Config, 'no buttons');
```